common/web/gesture-recognizer/src/engine/headless/touchpointCoordinator.ts

```diff
@@ -223,7 +223,21 @@ export class TouchpointCoordinator<HoveredItemType, StateToken=any> extends Even
     const selector = this.currentSelector;
 
     touchpoint.setGestureMatchInspector(this.buildGestureMatchInspector(selector));
-    this.emit('inputstart', touchpoint);
+
+    // If there's an error in code receiving this event, we must not let that break the flow of
+    // event input processing here!
+    try {
+      this.emit('inputstart', touchpoint);
+    } catch (err) {
+      console.error(err);
+    }
```

Comment:
AND log to sentry

Reply:
... this _is_ the log.  We don't directly call Sentry from Web; the mobile apps capture all `console.error` and `console.warn` logging _as_ Web's Sentry logs.

Reply:
Gotcha. Do we get stack capture with this?